src/main/java/edu/umich/ctools/sectionsUtilityTool/SISPollingThread.java

```diff
@@ -206,7 +208,7 @@ public void sendEmailReport(SISDataHolderForEmail data, String apiResp) {
 			M_log.info(String.format("SIS request %s of type %s for course %s Finished! Sending email report  " ,
 					data.getPollingId(),data.getUploadType(),data.getCourseId()));
 			Transport.send(message);
-		} catch (MessagingException | JSONException e) {
+		} catch (MessagingException | JSONException | IOException e) {
 			M_log.error(String.format(errmsg, data.getCourseId(), e.getMessage()));
 		} catch (Exception e) {
```

Comment:
You may want to add an extra string so the log can be grepped and the general exceptions that should have a specific handler are easy to find.  Maybe prefix with "UNEXPECTED: "?